app/components/contributors/edit_component.html.erb

```diff
@@ -17,16 +17,16 @@
                                                   action: 'click->contributors#contributorTypeOrganizationSelected' }) %>
   <% end %>
 
-  <%= render Elements::Forms::SelectFieldComponent.new(form:, field_name: :person_role,
+  <%= render Elements::Forms::SelectFieldComponent.new(form:, field_name: :person_role, label_classes: 'fw-bold',
                                                        options: PERSON_ROLES, label: t('works.edit.fields.contributors.role.label'),
                                                        data: { contributors_target: 'selectPersonRole' }) %>
-  <%= render Elements::Forms::SelectFieldComponent.new(form:, field_name: :organization_role,
+  <%= render Elements::Forms::SelectFieldComponent.new(form:, field_name: :organization_role, label_classes: 'fw-bold',
                                                        options: ORGANIZATION_ROLES, label: t('works.edit.fields.contributors.role.label'),
                                                        data: { contributors_target: 'selectOrganizationRole' }) %>
 
   <%# Contributor person name/orcid fields %>
-  <div data-contributors-target="personName">
-    <%= render Elements::Forms::LabelComponent.new(form:, field_name: :with_orcid, label_text: t('works.edit.fields.contributors.orcid_or_name.label')) %><br>
+  <fieldset data-contributors-target="personName">
+    <legend class="form-label fw-bold"><%= t('works.edit.fields.contributors.orcid_or_name.label') %></legend>
```

Comment:
Pull fieldset out into a component?

Reply:
I've got some concerns about component proliferation, so I might prefer delaying componentization until we have a need for a fieldset component elsewhere.

Reply:
Looks like it is used in at least 2 places.